airbyte-integrations/connectors/source-mailersend/source_mailersend/mailersend.yaml

```diff
@@ -0,0 +1,70 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["data"]
+  requester:
+    url_base: "https://api.mailersend.com/v1"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_token'] }}"
+    request_options_provider:
+      request_parameters:
+        limit: "100"
+  stream_slicer:
```

Comment:
@girarda , I'm trying to add stream slicer support but I'm getting this error:
```
{"message": "Something went wrong in the connector. See the logs for more details.", "internal_message": "invalid literal for int() with base 10: '2022-11-06'", "stack_trace": "Traceback (most recent call last):\n  File \"/Users/cirdes/Workspace/airbyte/airbyte-integrations/connectors/source-mailersend/main.py\", line 13
```

The api should be called like this:
https://api.mailersend.com/v1/activity/{domain_id}/?date_from=1667746844
date_from is the request_parameters and it's a Unix Timestamp (integer)

in spec.yml I have start_date
```
start_date:
  type: string
  description: Timestamp is assumed to be UTC. Must be lower than date_to.
  examples:
    - "2022-11-06"
```
And I'm using "2022-11-06"  as start_date.